src/observability/summarizer.go

```diff
@@ -125,6 +125,9 @@ func convertSysLogToSysSummaryMap(syslogs []*pb.Alert) {
 		sysSummary.Severity = syslog.Severity
 		sysSummary.PolicyName = syslog.PolicyName
 		sysSummary.Deployment = ""
+		sysSummary.Owner.Ref = syslog.Owner.Ref
+		sysSummary.Owner.Name = syslog.Owner.Name
+		sysSummary.Owner.Namespace = syslog.Owner.Namespace
 
 		for _, d := range deployments {
```

Comment:
@AmanKumar2696 If we are receiving all details from owner reference struct. Then kindly remove the client-go calls to get the deployment data ( i.e code from line 132 to 137)